tools/vagrant/README.md

```diff
@@ -79,6 +79,8 @@ wsk -i action invoke /whisk.system/samples/echo -p message hello --blocking --re
 configure the CLI with new values for __apihost__, __namespace__, and __auth__ key.
  
 ### Use the wsk CLI inside the VM
+Inside the VM the CLI command `wsk` points to `$HOME/bin/wsk` this is a wrapper script that uses `$OPENWHISK_HOME/bin/go-cli/linux/amd64/wsk` and the flag `-i` to be able to connect insecurly.
```

Comment:
For your convenience, a `wsk` wrapper is provided inside the VM which delegates CLI commands to ... and adds the `-i` parameter that is required for insecure access to the local OpenWhisk deployment.